src/app/loadout-builder/process/mappers.ts

```diff
@@ -82,12 +83,19 @@ export function mapDimItemToProcessItem({
     ? _.sumBy(modsForSlot, (mod) => mod.plug.energyCost?.energyCost || 0)
     : 0;
 
+  const thisIsArtifice = isArtifice(dimItem);
+  // as of TFS, [relevant, modern] exotics can use artifice stat mods, if the user pays to enhance the armor
+  const spoofArtifice =
+    !thisIsArtifice &&
+    dimItem.isExotic &&
```

Comment:
Exotic armor 1.0 vs. 2.0? Maybe this can be a helper function `isAssumedArtifice(DimItem, ArmorEnergyRules)`?

Reply:
does LO support 1.0 anymore?

Reply:
I don't believe it does.

Reply:
LO doesn't, but Loadouts themselves do support Armor 1.0 and may include them in mod assignment. If this ever assigns an artifice mod to an exotic 1.0 that can't have an artifice slot, `pickPlugPositions` can crash the app later.